### ISSUE TITLE
Create plan for the gh-dev web site

### ISSUE THREAD

**Issue by lcabrini** [osdev-gh/osdev-gh-www#1]
We need to decide what the purpose of the site is, and how it should be organized. This issue will probably spawn off several other issues.

**Comment by laudb** [comment]
i agree but I think since the website is important but not the prime focus may I propose this minimalist approach for start:

1) an overview of the project,
2) our timeline & milestones,
3) and there was something like a contribution board from the issue
we could do this on github pages too
plus, i sense we'll need to conclude on our naming else we'll have 'placeholder'OS

**Comment by lcabrini** [comment]
Good points. I was kind of hoping that you were ready to lead this sub-project, @laudb .

I think eventually we are going to have more web sites. I think osdev-gh is going to be a site that has member profile pages, information about us, schedules and links to how to get in touch. Also, all the papers and other things go up here. Thus, the first site should be more of a community site.

When we get working on our first OS, we should create a sub-domain for that, since that becomes a product we have created, or at least are in the process of creation.

What are your thoughts on this?

**Comment by laudb** [comment]
oh definitely :)

alright I initially was thinking something more project scoped but it also easily becomes a sub-domain to osdev-gh site, the community site, especially as more projects will also follow, so that works for me.  

so the community website basically has:
 - something about us
 - members section, with individual pages bearing contact information
 - i think a list of all projects which links to out project websites   

@lcabrini could you give further detail on this:

> Also, all the papers and other things go up here.

**Comment by lcabrini** [comment]
Yes, that looks perfect. With papers and other things I mean that we will be producing a lot of documentation as we are learning and studying. As it is one of the goals of the website that the members should get recognition for the work they have done all these artifacts (papers, tutorials, code, etc.) should somehow be linked to the member or members who worked on them. In a way, the member's profile page should be something they can put on their CV.

**Comment by lcabrini** [comment]
And we shouldn't forget to link to our github team page, and any social pages that we may set up.

**Comment by laudb** [comment]
okay so one more section which features uploads by various members be it documentation, tutorials, etc. 
yeah, my thoughts were to make the member page (or profile page) have a list of all activity respective to that member, I hope that's inline?

**Comment by lcabrini** [comment]
That's perfect. Among other goals, this should be a chance for our members to be able to showcase work that they have done. This could be useful for employment, promotion, etc. And we do want our members to be successful, right?

**Comment by lcabrini** [comment]
You have quite a number of people who showed interest in helping out on the website, so you might want to introduce it, maybe on whatsapp. Ask @Prince-linux to set up a web team within our group here on github and give you admin to it. You are now the WWW Overlord.

**Comment by rexfordnyrk** [comment]
With all this said I thing drupal can help us achieve this especially starting from the fact that it is a community based site. so far as I know redhat, and some other opensource community projects have their community site based on drupal. We can start with the base distribution [Drupal Commons](https://www.drupal.org/project/commons)
Then modify it completely from presentation to functionality most of which come out of the box
.

**Comment by lcabrini** [comment]
@rexfordnyrk let's discuss this at length in #2 instead.

**Comment by rexfordnyrk** [comment]
sorry just saw that one

**Comment by lcabrini** [comment]
That's ok. :)
